radio/src/targets/taranis/CMakeLists.txt

```diff
@@ -160,6 +160,12 @@ elseif(PCB STREQUAL X7)
     set(BLUETOOTH YES)
     add_definitions(-DBLUETOOTH_PROBE)
     set(ROTARY_ENCODER YES)
+    if(DEFINED PCBREV_HARDCODED)
```

Comment:
Missing comment from the original PR:
# This *can* be used to prevent messing with PA14 (SWCLK), so stlink can be used
# without interruptions.